.storybook/preview.js

```diff
@@ -1,4 +1,63 @@
 import {addons} from '@storybook/addons'
+import {ThemeProvider} from '../src/ThemeProvider'
+import {themeGet} from '../src'
```

Comment:
ThemeProvider is also exported from `src/index.ts` so we can combine these imports:

```suggestion
import {ThemeProvider, themeGet} from '../src'
```